addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -174,7 +174,7 @@ export class MassMailingHtmlField extends HtmlField {
 
         this.wysiwyg.odooEditor.observerFlush();
         this.wysiwyg.odooEditor.historyReset();
-        this.wysiwyg.$iframeBody.addClass('o_mass_mailing_iframe');
+        this.wysiwyg.$iframeBody.addClass('o_mass_mailing_iframe h-100');
```

Comment:
Not a fan of locking this to such a short height, can we keep the previous behaviour of resizing the div to allow the whole email to fit?

Reply:
Hello @thiryr , thanks for the review. I restored the default behaviour but now the snippet menu is not acting great on larger screens, as you said (is not fixed and thus leaves a white space at the bottom when scrolling the main div). Can you help me to fix that ? Thanks !

![Screenshot 2023-07-25 at 09 55 55](https://github.com/odoo/odoo/assets/110090660/1385b0ba-4b4f-4b21-80c9-c671cadbd498)

Reply:
Also added a fix for the alerts 👍

before:
![Screenshot 2023-07-25 at 10 05 56](https://github.com/odoo/odoo/assets/110090660/1118998b-0ebe-4a87-b22d-0d3f5ef2ad0d)
after: 
![Screenshot 2023-07-25 at 10 05 18](https://github.com/odoo/odoo/assets/110090660/b6ee1494-6d59-4fe8-8a09-f92e435a7846)

Reply:
@anso-odoo hey
Yep, we just need to make sure the size and position is updated on scrolling the form sheet instead of only the full content of the page. I think just looking for the form sheet should be alright for now so in `MassMailingFullWidthViewController` in `onMounted` we can just add `$('.o_form_sheet_bg').on('scroll.repositionMailingEditorSidebar', throttledRepositionSidebar);`

Since we use the same identifier as for the scrolling on content, it should already be unmounted so that's all :)
Don't remove the other one, as it's still needed for the landscape view.


Let me know if there was any other issue with keeping that component but it seems to work ok to me

Reply:
Looks like it works fine, thanks a lot !